tests/test_fixtures.py

```diff
@@ -158,7 +158,7 @@ def get_info(self, retry=30):
                                           timeout=60, stdout=subprocess.PIPE, env=self._run_env)
             if info_exe_res.returncode == 0 and \
                     str(info_exe_res.stdout).find('Provisioned on server: yes') != -1 and \
-                    str(info_exe_res.stdout).find('Current primary ecu running version:') != -1:
+                    str(info_exe_res.stdout).find('Current Primary ECU running version:') != -1:
```

Comment:
That's another proof that it's an antipattern to use raw string output as an input for another software component. aktualizr-info should be able to output the status info in a structured way - json, xml, yaml, etc. 
Most likely this change will require adjustment in OTF tests, too.

Reply:
Sorry, how would reformatting into xml or json have changed this in any way? Unless the strings are stored as constants in a shared place, you still have to change the string you are matching in both places.

That said, I'm not opposed to having all aktualizr-info output in json or another format, albeit probably with an optional flag. There might even be a ticket for it somewhere.

Reply:
> Sorry, how would reformatting into xml or json have changed this in any way?

I am not talking about just wrapping the text currently output by aktualizr-info into json or xml. First of all, it's about defining a data model and then corresponding representation of it in json. For example, in yaml it would be something like the following. I understand, that you might say that the yaml key names also can be changed but log messages are something that changes much more often compared to a key definition.

`
- device
    id: <>
    provisioned: true
    version: <>

    - ecu
         serial: <serial>
         hw-id: <>
         type: primary

   - ecu
        serial: <serial>
        hw-id: <>
        type: secondary
`